yarn-project/aztec-cli/README.md

```diff
@@ -232,9 +236,32 @@ Example usage:
 aztec-cli get-contract-data 0x123456789abcdef123456789abcdef12345678
 ```
 
+### register-public-key
+
+Register an account's public key to the RPC server
+
+Syntax:
+
+```shell
+aztec-cli register-public-key [options]
+```
+
+Options:
+
+- `-a, --address <aztecAddress>`: The account's Aztec address.
+- `-p, --public-key <publicKey>`: 'The account public key.'
+- `-pa, --partial-address <partialAddress`: 'The partially computed address of the account contract.'
```

Comment:
I would link here [the docs Santiago wrote](https://github.com/AztecProtocol/aztec-packages/blob/master/docs/docs/aztec/developer/wallet-providers/keys.md#addresses-partial-addresses-and-public-keys) about partial address etc. I feel like these terms are super confusing and I would have no idea what's going on here.